xonsh/prompt/vc.py

```diff
@@ -163,9 +168,11 @@ def current_branch():
     return branch or None
 
 
-def _get_exit_code(cmd):
+def _get_exit_code(cmd, additional_env=None):
```

Comment:
It seems like you can just update the denv and not need to add an argument to this function

Reply:
That is, all call sites of `_get_exit_code` seem to pass in the same env here